pimd/pim_cmd.c

```diff
@@ -6861,12 +6870,33 @@ DEFUN (no_ip_pim_packets,
 	return CMD_SUCCESS;
 }
 
-DEFUN (ip_pim_v6_secondary,
-       ip_pim_v6_secondary_cmd,
-       "ip pim send-v6-secondary",
-       IP_STR
-       "pim multicast routing\n"
-       "Send v6 secondary addresses\n")
+DEFPY(igmp_group_watermark, igmp_group_watermark_cmd,
```

Comment:
all the functions in this code have `DEFUN (` le'ts follow suit on how things are formated.

Reply:
clang formatter did it. Also has rearranged help string lines here. Will change it manually now.

Reply:
Yes, clang-format doesn't understand our `DEFUN/DEFPY` conventions, there's a blurb on this in the docs

Reply:
changed to right format